hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceWALReader.java

```diff
@@ -276,6 +274,8 @@ public Path getCurrentPath() {
   private boolean checkQuota() {
     // try not to go over total quota
     if (totalBufferUsed.get() > totalBufferQuota) {
+      LOG.warn("Can't read more edits from WAL as buffer usage {}B exceeds limit {}B",
```

Comment:
Nit: Worth log for which peer the given source reader is sleeping?

Reply:
Sure, that's easy to add.